data/sql/updates/pending_db_world/rev_1667605127641040300.sql

```diff
@@ -0,0 +1,9 @@
+--
+UPDATE creature_template_locale SET Title='Gemischtwaren' WHERE entry=2808 AND locale='deDE';
```

Comment:
```suggestion
UPDATE `creature_template_locale` SET `Title`='Gemischtwaren' WHERE `entry`=2808 AND `locale`='deDE';
```
Hi, please add backticks around all column names.